packages/block-library/src/latest-posts/edit.js

```diff
@@ -363,20 +363,36 @@ class LatestPostsEdit extends Component {
 							[ `align${ featuredImageAlign }` ]: !! featuredImageAlign,
 						} );
 
-						const postExcerpt =
+						const needsReadMore =
 							excerptLength <
 								excerpt.trim().split( ' ' ).length &&
-							post.excerpt.raw === ''
-								? excerpt
+							post.excerpt.raw === '';
+
+						const getReadMoreLink = ( excerptText, link ) => {
+							return (
+								<>
+									{ excerptText }
+									&nbsp;&hellip;&nbsp;
```

Comment:
Thinking about i18n, this is the kind of thing that we may leave in the hands of translators. Similarly to how we don't hard-code punctuation around plain strings, in this case we might prefer to let translators define how to best truncate for their locale, whether with horizontal ellipsis or something else.

Reply:
Yes, good one, I didn't realize there are multilingual options for that truncation char but there are. Since we don't want it as part of the link it is now translated on its own, and since it's just an ellipse char I also added a translators comment.